openfisca_aotearoa/tests/social_security/accommodation_supplement/2018/exclusion.yaml

```diff
@@ -1,8 +1,9 @@
 - name: >
     Social Security Act 2018 -
-    65 Accommodation Supplement -
-    66 Social housing exclusion
-    (a) Rent
+    Subpart 10 Accommodation Supplement -
+    66  Social housing exclusion -
+    (1) Social housing exclusion -
+        (a) Rent
   period: 2018-11-26
   input:
     accommodation_type:
```

Comment:
What is the purpose of `under_the_sun`? It's not a valid AccommodationType value, I note further in this file is `under_the_moon`. I'm not entirely sure why OpenFisca accepts this.